### ISSUE TITLE
add hints 

### ISSUE THREAD

**Issue by jellevankerk** [where-the-flag-is-this/where-the-flag-is-this.github.io#14]
It would be nice that we could pres a hint button the get for example a hint about the country. 
e.g continent, fact about the country or population

**Comment by wagenrace** [comment]
I wanted to add a selection for contient, could we reuse that for a hint?

**Comment by jellevankerk** [comment]
yes probably, but we need also add the continent to the metadata

**Comment by wagenrace** [comment]
I am on it with the selection tool, sorry did not had a lot of time/energy these days
